lib/libc/musl/src/math/i386/floor.s

```diff
@@ -1,15 +1,3 @@
-.global floorf
```

Comment:
Need to add a `/* zig patch: ... */` comment to files that are modified like this so the change is kept on musl updates.